src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -62,9 +62,40 @@ public void storeLocation(Location location) {
         locationEntity.setProperty("name", location.getName());
         locationEntity.setProperty("description", location.getDescription());
 
+        if(location.getImageUrl() != null) {
+            locationEntity.setProperty("imageUrl", location.getImageUrl());
+        }
+
         datastore.put(locationEntity);
     }
 
+    public List<Location> getLocations() {
```

Comment:
Leave getLocations and readLocation for the person implementing the location feed.